CMakeLists.txt

```diff
@@ -224,7 +224,8 @@ else()
     set(GIT_BRANCH "UNKNONW")
 endif()
 
-list(APPEND GIT_BUILD_INFO_DEFINES
+add_library(git-info INTERFACE)
+target_compile_definitions(git-info INTERFACE
```

Comment:
Do jpscore and jpsreport use these lines together?

Reply:
So it works like this:
in line 227 a new "interface" library is defined that allows to propagate compilation flags, include paths, preprocessor definitions etc. by linking to this library
So jpsreport and jpscore depend on this library (see the respective target_link_libraries calls) and the defines will be set during the compilation of the source files.
TL;DR; yes :)

Reply:
Thanks.